lab_solutions/lab01_qsort.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Patrick Nanys <patrick.nanys2000@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+
+def swap(l, idx1, idx2):
+    temp = l[idx1]
+    l[idx1] = l[idx2]
+    l[idx2] = temp
+
+
+def qsort(l, start, end):
```

Comment:
`qsort` should only take one mandatory argument, the list.

Reply:
Sorry, I accidentally added a previous version of my code.